lib/imapoptions

```diff
@@ -549,7 +549,7 @@ Blank lines and lines beginning with ``#'' are ignored.
 { "backup_retention", "7d", DURATION, "3.12.0", "3.12.0" }
 /* Deprecated. No longer used. */
 
-{ "backup_db", "twoskip", STRINGLIST("skiplist", "sql", "twoskip", "zeroskip"), "3.12.0", "3.12.0" }
+{ "backup_db", "twoskip", STRINGLIST("skiplist", "sql", "twoskip"), "3.12.0", "3.12.0" }
```

Comment:
This case is interesting, because the option is already deprecated (since 3.12.0), but we're changing it again (by necessity I think).

I think perhaps you should change the first version number (which is the `last_modified` field) to "UNRELEASED", to reflect that it has been modified; but leave the second version number (the `deprecated_since` field) as "3.12.0", since that was when it was deprecated and that fact hasn't changed.

If that doesn't work, then I guess leave both version numbers as "3.12.0".

Reply:
done. please recheck.